apps/sms/views/shared/js/activity_handler.js

```diff
@@ -380,13 +380,16 @@ var ActivityHandler = {
         }
 
         function continueWithNotification(sender, body) {
-          var title = sender;
+          var titlePromise;
+
           if (Settings.hasSeveralSim() && message.iccId) {
             var simName = Settings.getSimNameByIccId(message.iccId);
-            title = _(
+            titlePromise = formatValue(
```

Comment:
one question: once we use formatValue do we still need to wait for l10n ready at startup?